moby/moby
# containerd integration: `docker diff`

**- What I did**

Implement `docker diff` with containerd store

**- How I did it**

Upstream:
- https://github.com/rumpl/moby/pull/91

**- How to verify it**

(run a container and create a file)
- `docker run -it --name hello alpine`
- `/ # echo world > world.txt && exit`

(verify `diff` works)
- `docker diff hello`

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**

<img width="573" alt="image" src="https://user-images.githubusercontent.com/70572044/217833596-91d46ba3-8469-4dc9-980c-adc6d80cc524.png">
